docs/guide/nameservice/02-messages.md

```diff
@@ -107,17 +107,17 @@ The `ignite scaffold message buy-name name bid` command creates and modifies sev
     syntax = "proto3";
     package cosmonaut.nameservice.nameservice;
 
-    // this line is used by starport scaffolding # proto/tx/import
+    // this line is used by ignite scaffolding # proto/tx/import
```

Comment:
these comments remain as is for now with "starport scaffolding", the code is not updated here

am I understanding correctly @ilgooz ?

Reply:
Yes, we shouldn't change this and similar ones.